crates/noirc_frontend/src/lexer/token.rs

```diff
@@ -325,23 +355,17 @@ pub enum Attribute {
     Builtin(String),
     Oracle(String),
     Deprecated(Option<String>),
-    Test { expect_failure: bool },
+    Test { scope: TestScope },
```

Comment:
Do you still want the record variant here or would it just be `Test(TestScope)`?

Reply:
I like this better, since scope is in the struct name